sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -41,8 +36,7 @@ import {
   UserActiveStatusAPIInDTO,
   InstitutionUserDetailAPIOutDTO,
   InstitutionUserLocationsAPIOutDTO,
-  UserRoleOptionAPIOutDTO,
-  InstitutionUserTypeAndRoleAPIOutDTO,
```

Comment:
As we removed all the referenced for the InstitutionUserTypeAndRoleAPIOutDTO, please remove them in the institution-user.dto
![image](https://user-images.githubusercontent.com/62901416/179098853-fe1b0468-58cc-483c-8701-83abd4137733.png)
![image](https://user-images.githubusercontent.com/62901416/179098853-fe1b0468-58cc-483c-8701-83abd4137733.png)

Reply:
Removed!